lib/internal/http2/compat.js

```diff
@@ -330,45 +306,97 @@ class Http2ServerResponse extends Stream {
     code |= 0;
     if (code >= 100 && code < 200)
       throw new RangeError('Informational status codes cannot be used');
-    if (code < 200 || code > 999)
+    if (code < 200 || code >= 600)
       throw new RangeError(`Invalid status code: ${code}`);
     state.statusCode = code;
   }
 
+  addTrailers(headers) {
+    for (var name of Object.keys(headers)) {
+      name = String(name).trim().toLowerCase();
+      var value = headers[name];
+      assertValidHeader(name, value);
+      var trailers = this[kTrailers];
+      if (!trailers)
+        trailers = this[kTrailers] = Object.create(null);
```

Comment:
Looking at a [jsPerf test case](https://jsperf.com/object-create-null-vs-literal/2), it seems `{}` is 10-20x faster than `Objet.create(null)`. Should change it to that, I guess.